website/docs/language/expressions/conditionals.mdx

```diff
@@ -78,3 +78,134 @@ a string.
 ```hcl
 var.example ? tostring(12) : "hello"
 ```
+
+## Examples
```

Comment:
Moving this all from the old "Custom Conditions" page I'm replacing, and it felt like a natural fit here!